bench/wtperf/misc.c

```diff
@@ -106,13 +106,13 @@ lprintf(const WTPERF *wtperf, int err, uint32_t level, const char *fmt, ...)
 }
 
 /*
- * backup_read --
- *     Read in a file, used mainly to measure the impact of backup on a single machine. Backup is
- *     usually copied across different machines, therefore the write portion doesn't affect the
- *     machine backup is performing on.
+ * backup_read - Read in a file, used mainly to measure the impact 
```

Comment:
Looks like the extra dash - got removed which caused clang format to mangle the comment. We want the `backup_read --` so that it is properly formatted.

Reply:
thanks! placed it in